tests/Tools/CodeHacking/CodeHacker.php

```diff
@@ -0,0 +1,529 @@
+<?php
+/**
+ * CodeHacker class file.
+ *
+ * @package WooCommerce/Testing
+ */
+
+//phpcs:disable WordPress.WP.AlternativeFunctions, WordPress.PHP.NoSilencedErrors.Discouraged
```

Comment:
let's not disable phpcs for entire hacks, if there is good reason to do so we should mention it.

Reply:
I prefer to ignore those in each line, but since this is an internal tool, sounds fine to remove in all instances.
But note that doing it in our code base isn't recommended, since we can use PHP CodeSniffer to detect those instances and help us write better code.